app/webpacker/src/stylesheets/_tables.scss

```diff
@@ -221,3 +221,60 @@ table.section-browser {
     //}
   }
 }
+
+.govuk-table--responsive {
+  @include govuk-media-query($until: tablet) {
+    caption, thead, tbody, th, td, tr {
+      display: block;
+    }
+    thead tr {
+      /* Hide table headers but keep them screenreader-accessible */
```

Comment:
What's a screenreader?

Reply:
An accessibility tool for users - it reads the screen contents out. You can override what they read out using aria tags but its easier if the original markup 'just works'